javascript/javascript3/week1/giphyindex.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="adjust">
+        <input type="text" id="searchText" value="search for gif">
+        <button >search</button><br><br>
+        <input type="text" id="number" value="Number of gifs">
+    </div>
+    <div id="center">
```

Comment:
Could be nice if reuse center2 instead of #center. Also, center is confusing, maybe find a better descriptive name.